From 7df635c9ac779b2c5feaf333160260b541514682 Mon Sep 17 00:00:00 2001
From: Cloud User <will@stackhpc.com>
Date: Thu, 14 Jun 2018 13:47:06 +0000
Subject: [PATCH] Change default user to centos

---
 rhel/cloud.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rhel/cloud.cfg b/rhel/cloud.cfg
index 986f241..675df1c 100644
--- a/rhel/cloud.cfg
+++ b/rhel/cloud.cfg
@@ -51,7 +51,7 @@ cloud_final_modules:
 
 system_info:
   default_user:
-    name: cloud-user
+    name: centos
     lock_passwd: true
     gecos: Cloud User
     groups: [wheel, adm, systemd-journal]
-- 
1.8.3.1

